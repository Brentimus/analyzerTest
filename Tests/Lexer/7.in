Array
Asm
Begin
Break
Case
Const
Constructor
Continue
Destructor
Do
Downto
Else
End
False
File
For
Function
Goto
If
Implementation
In
Inline
Interface
Label
Nil
Not
Object
Of
Operator
Packed
Procedure
Program
Record
Repeat
Set
Shl
Shr
String
Then
To
True
Type
Unit
Until
Uses
Var
While
With
As
Class
Constref
Dispose
Except
Exit
Exports
Finalization
Finally
Inherited
Initialization
Is
Library
New
On
Out
Property
Raise
Self
Threadvar
Try
Absolute
Abstract
Alias
Assembler
Cdecl
Cppdecl
Default
Export
External
Forward
Generic
Index
Local
Name
Nostackframe
Oldfpccall
Override
Pascal
Private
Protected
Public
Published
Read
Register
Reintroduce
Safecall
Softfloat
Specialize
Stdcall
Virtual
Write