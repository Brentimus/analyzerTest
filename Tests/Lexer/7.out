1  1  Keyword  ARRAY  Array
2  1  Keyword  ASM  Asm
3  1  Keyword  BEGIN  Begin
4  1  Keyword  BREAK  Break
5  1  Keyword  CASE  Case
6  1  Keyword  CONST  Const
7  1  Keyword  CONSTRUCTOR  Constructor
8  1  Keyword  CONTINUE  Continue
9  1  Keyword  DESTRUCTOR  Destructor
10  1  Keyword  DO  Do
11  1  Keyword  DOWNTO  Downto
12  1  Keyword  ELSE  Else
13  1  Keyword  END  End
14  1  Keyword  FALSE  False
15  1  Keyword  FILE  File
16  1  Keyword  FOR  For
17  1  Keyword  FUNCTION  Function
18  1  Keyword  GOTO  Goto
19  1  Keyword  IF  If
20  1  Keyword  IMPLEMENTATION  Implementation
21  1  Keyword  IN  In
22  1  Keyword  INLINE  Inline
23  1  Keyword  INTERFACE  Interface
24  1  Keyword  LABEL  Label
25  1  Keyword  NIL  Nil
26  1  Operator  Not  Not
27  1  Keyword  OBJECT  Object
28  1  Keyword  OF  Of
29  1  Keyword  OPERATOR  Operator
30  1  Keyword  PACKED  Packed
31  1  Keyword  PROCEDURE  Procedure
32  1  Keyword  PROGRAM  Program
33  1  Keyword  RECORD  Record
34  1  Keyword  REPEAT  Repeat
35  1  Keyword  SET  Set
36  1  Operator  Shl  Shl
37  1  Operator  Shr  Shr
38  1  Keyword  STRING  String
39  1  Keyword  THEN  Then
40  1  Keyword  TO  To
41  1  Keyword  TRUE  True
42  1  Keyword  TYPE  Type
43  1  Keyword  UNIT  Unit
44  1  Keyword  UNTIL  Until
45  1  Keyword  USES  Uses
46  1  Keyword  VAR  Var
47  1  Keyword  WHILE  While
48  1  Keyword  WITH  With
49  1  Keyword  AS  As
50  1  Keyword  CLASS  Class
51  1  Keyword  CONSTREF  Constref
52  1  Keyword  DISPOSE  Dispose
53  1  Keyword  EXCEPT  Except
54  1  Keyword  EXIT  Exit
55  1  Keyword  EXPORTS  Exports
56  1  Keyword  FINALIZATION  Finalization
57  1  Keyword  FINALLY  Finally
58  1  Keyword  INHERITED  Inherited
59  1  Keyword  INITIALIZATION  Initialization
60  1  Keyword  IS  Is
61  1  Keyword  LIBRARY  Library
62  1  Keyword  NEW  New
63  1  Keyword  ON  On
64  1  Keyword  OUT  Out
65  1  Keyword  PROPERTY  Property
66  1  Keyword  RAISE  Raise
67  1  Keyword  SELF  Self
68  1  Keyword  THREADVAR  Threadvar
69  1  Keyword  TRY  Try
70  1  Keyword  ABSOLUTE  Absolute
71  1  Keyword  ABSTRACT  Abstract
72  1  Keyword  ALIAS  Alias
73  1  Keyword  ASSEMBLER  Assembler
74  1  Keyword  CDECL  Cdecl
75  1  Keyword  CPPDECL  Cppdecl
76  1  Keyword  DEFAULT  Default
77  1  Keyword  EXPORT  Export
78  1  Keyword  EXTERNAL  External
79  1  Keyword  FORWARD  Forward
80  1  Keyword  GENERIC  Generic
81  1  Keyword  INDEX  Index
82  1  Keyword  LOCAL  Local
83  1  Keyword  NAME  Name
84  1  Keyword  NOSTACKFRAME  Nostackframe
85  1  Keyword  OLDFPCCALL  Oldfpccall
86  1  Keyword  OVERRIDE  Override
87  1  Keyword  PASCAL  Pascal
88  1  Keyword  PRIVATE  Private
89  1  Keyword  PROTECTED  Protected
90  1  Keyword  PUBLIC  Public
91  1  Keyword  PUBLISHED  Published
92  1  Keyword  READ  Read
93  1  Keyword  REGISTER  Register
94  1  Keyword  REINTRODUCE  Reintroduce
95  1  Keyword  SAFECALL  Safecall
96  1  Keyword  SOFTFLOAT  Softfloat
97  1  Keyword  SPECIALIZE  Specialize
98  1  Keyword  STDCALL  Stdcall
99  1  Keyword  VIRTUAL  Virtual
100  1  Keyword  WRITE  Write
100  6  Eof